app/controllers/jobs_controller.rb

```diff
@@ -41,8 +29,8 @@ def index
     end
 
     # Get the complete lists of live and dead jobs from the server
-    raw_live_jobs = getCurrentJobs
-    raw_dead_jobs = getDeadJobs
+    raw_live_jobs = TangoClient.tango_jobs()["jobs"]
```

Comment:
Could we check the statusId in TangoClient, and return a null or empty object if it's a failure status?  I think part of the reason we want this TangoClient is so that Tango details are abstracted away from Autolab, so that multiple backends can be supported.  So to me, it makes sense that the jobs function would always return an object describing the jobs.
That said, this is just a step toward a fully-abstracted TangoClient, so if you don't agree or don't want that for this commit, I understand.